index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>JavaScript Basics</title>
+  </head>
+  <body>
+    <script src="script.js"></script>
```

Comment:
I get an error when running this,
<img width="492" alt="image" src="https://github.com/user-attachments/assets/1e095bc7-8b7d-47c3-a41c-b1d56371d610">
It looks like it should be
```suggestion
    <script src="scripts.js"></script>
```